seml/main.py

```diff
@@ -70,57 +67,83 @@ def main():
+                 "Experiments that were staged together have the same batch_id.")
+    parser_start_launch_parent.add_argument(
+            '-f', '--filter-dict', type=json.loads,
+            help="Dictionary (passed as a string, e.g. '{\"config.dataset\": \"cora_ml\"}') to filter "
+                 "the experiments by.")
+    parser_start_launch_parent.add_argument(
+            '-o', '--output-to-console', action='store_true',
+            help="Print output to console.")
+    parser_start_launch_parent.add_argument(
+            '-nf', '--no-file-output', action='store_true',
+            help="Print output to console.")
+    parser_start_launch_parent.add_argument(
+            '-wg', '--worker-gpus', type=str,
+            help="The IDs of the GPUs used by the local worker. Will be directly passed to CUDA_VISIBLE_DEVICES. "
+                 "Has no effect for Slurm jobs.")
+    parser_start_launch_parent.add_argument(
+            '-wc', '--worker-cpus', type=int,
+            help="The number of CPU cores used by the local worker. Will be directly passed to OMP_NUM_THREADS. Has no "
+                 "effect for Slurm jobs.")
+    parser_start_launch_parent.add_argument(
+            '-we', '--worker-environment-vars', type=json.loads,
+            help="Further environment variables to be set for the local worker. Has no effect for Slurm jobs.")
+
     parser_start = subparsers.add_parser(
             "start",
+            parents=[parser_start_launch_parent],
             help="Fetch staged experiments from the database and run them (by default via Slurm).")
     parser_start.add_argument(
             '-l', '--local', action='store_true',
             help="Run the experiments locally (not via Slurm).")
     parser_start.add_argument(
-            '-n', '--num-exps', type=int, default=-1,
-            help="Only start the specified number of experiments.")
-    parser_start.add_argument(
-            '-u', '--unobserved', action='store_true',
-            help="Run the experiments without Sacred observers (no changes to the database). "
-                 "This also disables output capturing by Sacred, facilitating the use of debuggers (pdb, ipdb).")
-    parser_start.add_argument(
-            '-pm', '--post-mortem', action='store_true',
-            help="Activate post-mortem debugging with pdb.")
+            '-nw', '--no-worker', action='store_true',
+            help="Do not launch a local worker after setting experiments' state to PENDING.")
     parser_start.add_argument(
             '-d', '--debug', action='store_true',
             help="Run a single experiment locally without Sacred observers and with post-mortem debugging. "
                  "This is equivalent to "
-                 "`--verbose --local --num-exps 1 --unobserved --post-mortem --output-to-console`.")
+                 "`--verbose --local --num-exps 1 --set_to_pending --post-mortem --output-to-console`.")
```

Comment:
The user can't use `set_to_pending`. Also, this is no longer equivalent since we removed the `unobserved` option and only imply it via `debug`. So this needs to be rephrased.

Reply:
See d8b23db. Better?

Reply:
Changed it slightly in 67627ae.